ansys/tools/repo_sync/repo_sync.py

```diff
@@ -35,48 +35,21 @@ def synchronize(
 
     # Create a temporary folder
     with tempfile.TemporaryDirectory() as temp_dir:
-        os.chdir(temp_dir)
```

Comment:
Using `cwd` for the subprocess calls instead of `chdir`, because removing the temporary directory may fail if there's still a process accessing it (at least on Windows).